template/src/components/ColorHeader/ColorHeader.js

```diff
@@ -1,5 +1,7 @@
 import { registerHtml, useState } from "tram-one"
 import "./ColorHeader.scss"
+import "core-js/stable";
+import "regenerator-runtime/runtime";
```

Comment:
these shouldn't be required here, polyfills should only be added to the top most file (which in this case would be index.js)